README.md

```diff
@@ -81,11 +87,23 @@ Additionally, you can send an options hash with the following keys:
 options = {
   template: "blog", # Will use one of the predefined templates and create Content Types, Assets and Entries
   json_template: "/path/to/template.json", # Will use the JSON file specified as a Template
+  mark_processed: false, # if true will mark all items as 'bootstrapProcessed' and will be avoided for update_space calls (doesnt affect create_space)
   trigger_oauth: true # if true will trigger OAuth process
 }
 Contentful::Bootstrap::CommandRunner.new.create_space("space_name", options)
 ```
 
+To Update an existing Space
+
+```ruby
+options = {
+  json_template: "/path/to/template.json", # Will use the JSON file specified as a Template
+  mark_processed: false, # if true will mark all items as 'bootstrapProcessed and will be avoided on future update_space calls
```

Comment:
I think we used to say "resources" instead of "items" — don't think we have a consensus, though 😢